### PR TITLE
Dockerize Anemometer

### PR DESCRIPTION
**Pull request by akuzminsky**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @akuzminsky has signed the CLA. Thanks for the pull request!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM centos:7
+MAINTAINER Box Open Source "ops@box.com"
```
Comment (geoffreyanderson) [inline Dockerfile]: should this be `oss@box.com`?

**Comment by geoffreyanderson** [comment]
@akuzminsky how's this compare to #153 ?

**Comment by akuzminsky** [comment]
@geoffreyanderson , two biggest differences:
1. Rewritten entypoint in python
2. #153 uses php image which doesn't have bcmath on some reason. I use plain centos7 image and install php + apache+other stuff.

**Pull request merged**